src/main/java/com/snowflake/kafka/connector/internal/streaming/StreamingClientProperties.java

```diff
@@ -90,6 +90,11 @@ public StreamingClientProperties(Map<String, String> connectorConfig) {
 
     // Override only if the streaming client properties are explicitly set in config
     this.parameterOverrides = new HashMap<>();
+    if (isIcebergEnabled) {
+      // TODO should be replaced by SnowflakeStreamingIngestClientFactory.builder.setIceberg(true)
+      // call in SNOW-1728002
+      this.parameterOverrides.put("enable_iceberg_streaming", "true");
+    }
```

Comment:
By the way the param is set in IcebergIngestionIT setUp.

Reply:
There are no enum for "enable_iceberg_streaming" in ingest-sdk.

Reply:
I found one. Please try: `ParameterProvider.ENABLE_ICEBERG_STREAMING`

Reply:
Additionally please remove the TODO comment and add `SNOW-1728002` to the PR name. This is everything that needs to be done in this scope.

Reply:
> I found one. Please try: ParameterProvider.ENABLE_ICEBERG_STREAMING

I had to update ingest-sdk for it.

> Additionally please remove the TODO comment and add SNOW-1728002 to the PR name. This is everything that needs to be done in scope of this JIRA.

Done

Reply:
When I used variable from ParameterProvider it fails to compile, I think ingest-sdk 3.0.0 is not yet available